index.html

```diff
@@ -1644,6 +1644,72 @@ <h2>
     </section>
   </section>
   </section>
+
+  <section>
+    <h2>Verification Algorithm</h2>
+    <p>
+This specification might be used with many different key discovery protocols.
+As such, discovery of verification keys is described in a different section of this document, 
+and is assumed to have succeeded prior to beginning the verification process.
+    </p>
+    <p>
+As a general rule, verifiers SHOULD strive to minimize the processing of untrusted data.
+This includes minimizing any processing of the protected header, unprotected header, or payload as part of the key discovery procedures.
+    </p>
+
+    <p>
+When verifying a credential or presentation secured with SD-JWT, the algorithm defined in 
+<a href="https://datatracker.ietf.org/doc/html/draft-ietf-oauth-selective-disclosure-jwt-06#name-verification-of-the-sd-jwt">
+  Verification of the SD-JWT
+</a> MUST be followed.
+    </p>
+
+    <p>
+When verifying a credential or presentation secured with COSE Sign 1, the algorithm defined in 
+<a data-cite="RFC9052#section-4.4">
+Signing and Verification Process
+</a> MUST be followed.
+    </p>
+
+    <p>
+After verification has succeeded, additional validation checks SHOULD be performed.
+    </p>
+
+  </section>
+
+  <section>
+    <h2>Validation Algorithm</h2>
+
+    <p>
+All claims expected for the <code>typ</code> MUST be present.
```

Comment:
Is there a specification somewhere that maps from `typ` codes to the expected set of claims?